includes/notes/class-wc-admin-note.php

```diff
@@ -485,7 +485,7 @@ public function clear_actions() {
 	public function add_action( $name, $label, $url = '', $status = self::E_WC_ADMIN_NOTE_ACTIONED, $primary = false ) {
 		$name    = wc_clean( $name );
 		$label   = wc_clean( $label );
-		$query   = esc_url( $url );
+		$query   = esc_url_raw( $url );
```

Comment:
https://codex.wordpress.org/Function_Reference/esc_url_raw

Reply:
Does this branch just need a rebase? https://github.com/woocommerce/woocommerce-admin/commit/4f199782b3ae87596bff20fd4fc34453392efc39